pulumi/Pulumi.local-grapl.yaml

```diff
@@ -0,0 +1,13 @@
+encryptionsalt: v1:KRH5XXw/Vv4=:v1:Mi1EOttYFIuFy8Zl:u/QoTfZuh9c5m6Jyw2L3mAKC+VNs8g==
+config:
+  aws:accessKey: test
+  aws:endpoints:
+    - dynamodb: http://localhost:4566
+      s3: http://localhost:4566
+      secretsmanager: http://localhost:4566
+      sqs: http://localhost:4566
+  aws:region: us-east-1
```

Comment:
Unrelated to your PR:
I actually wonder if we should have our default region, locally, be:
- a non-default region (like eu-west-1)
- a made up one (like local-region-1)

i feel like we'd do a better job of ensuring all our code is correctly region-aware

Reply:
IIRC localstack is very particular about regions

Reply:
The documentation suggests we can set `DEFAULT_REGION` when starting Localstack and set it to whatever we want. I can try that and see if everything still works. If so, it could be a nice way to ensure that we don't accidentally bake in `us-east-1` assumptions in our services.

Reply:
I think this may work, but we do have `us-east-1` in a handful of places in the code, and not all of it is testing / local grapl code.

I'll leave the code as-is in this PR, but will file an issue to weed this out of the codebase.